changelog/update-update-husky

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: dev
```

Comment:
Will this generate an entry in the changelog? If so, is that desirable? Some dependency upgrades we might want to appear in the changelog though....thoughts?

Reply:
It will show up, yes. This can be seen in the below PR and respective change in the changelog.

* https://github.com/Automattic/woocommerce-payments/pull/4857/files#diff-4542d9055e8f89ac580d7b0026e77635e7f647bea4bd0e1c217fbc4c170778e0R1
* https://github.com/Automattic/woocommerce-payments/blob/develop/changelog.txt#L26

Having the updates listed in the changelog gives us an easy reference for when the dependency was updated, but it's not necessarily needed. I added the changelog entry due to it's a standard thing we do. I don't think it's going to hurt anything, the repo itself is public, which makes the dependencies in `package.json` public.

Reply:
Yeah, on second thought I like that it'll be listed as a `dev` dependency